cameraview/src/main/java/com/otaliastudios/cameraview/video/SnapshotVideoRecorder.java

```diff
@@ -141,7 +143,14 @@ public void onRendererFrame(@NonNull SurfaceTexture surfaceTexture, int rotation
                 case H_264: videoType = "video/avc"; break; // MediaFormat.MIMETYPE_VIDEO_AVC:
                 case DEVICE_DEFAULT: videoType = "video/avc"; break;
             }
-            String audioType = "audio/mp4a-latm";
+            String audioType = "";
+            switch (mResult.audioCodec) {
+                case AAC:
+                case HE_AAC:
+                case AAC_ELD:
+                case DEVICE_DEFAULT:
```

Comment:
Opting to use "mp4a-latm" instead of "audio/3gpp" for `AudioCodec.DEVICE_DEFAUT` here because using it's [technically correct value](https://github.com/natario1/CameraView/blob/master/cameraview/src/main/java/com/otaliastudios/cameraview/video/FullVideoRecorder.java#L122) would change the default behaviour of the class and therefore require all developers to upgrade.

Similar to how `DEVICE_DEFAULT` in the video codec block above is defaulted to "video/avc" even though that may not be a true representation of the `DEVICE_DEFAULT` value.